modules/support.lua

```diff
@@ -150,6 +159,9 @@ selectPreferred = function(searchMethod)
 			click(game.SUPPORT_UPDATE_CLICK)
 			wait(1)
 			click(game.SUPPORT_UPDATE_YES_CLICK)
+			while NeedsToRetry() do
```

Comment:
I think the code for retrying should be moved to a different module, unless you remove `local` from the function declaration, but that would look dirty.

![image](https://user-images.githubusercontent.com/4316326/74674762-d215b580-5190-11ea-85c0-8709935cdf5d.png)

Reply:
Not sure what the issue was to be honest. It worked fine for me when I tested it myself. But regardless, I made some changes so that it is in now in the Game module as a global function. Check it out and see if it works now.